zapai/example/main.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"net/http"
+	"os"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
```

Comment:
is this an example of just `zap`? don't see references to `zapai`

Reply:
If I'm reading this right, the AI sink is registered in an init function of the zapai package (`zapai/sink.go`) which I (having no experience with zap) assume means we just need to `import _ "github.com/Azure/azure-container-networking/zapai"` to get the side-effect?

Reply:
i accidentally dropped the ai zapcore stash, i have updated this file with example usages for the standard json logger, logfmt, and zapai

Reply:
> the AI sink is registered in an init function of the zapai package

this registers our[ ai sink factory method](https://github.com/Azure/azure-container-networking/pull/1158/files#diff-4482faabbd028f6614b30a0970ef8c3695da11e46b5f70ff0c214a9dd24df620R37) with the zap sink machinery, so that we can `zap.Open("appinsights://xxx")` (the URI should be generated [by the method attached to the config struct](https://github.com/Azure/azure-container-networking/pull/1158/files#diff-4482faabbd028f6614b30a0970ef8c3695da11e46b5f70ff0c214a9dd24df620R82)) and we get a zapai.Sink.